fix/stuck-instruments.lua

```diff
@@ -0,0 +1,65 @@
+-- Fixes instruments that never got played during a performance
+
+local help = [====[
+
+fix/stuck-instruments
+=====================
+
+Fixes instruments that were picked up for a performance, but were instead
+simulated and are now stuck permanently in a job that no longer exists.
+
+This works around the issue encountered with :bug:`9485`, and should be run
+if you notice any instruments lying on the ground that seem to be stuck in a
+job.
+
+Run ``fix/stuck-instruments -n`` or ``fix/stuck-instruments --dry-run`` to
+list how many instruments would be fixed without performing the action.
+
+]====]
+
+
+function fixInstruments(args)
+    local dry_run = false
+    local fixed = 0
+    for _, arg in pairs(args) do
+        if args[1]:match('-h') or args[1]:match('help') then
```

Comment:
use the `argparse` module for commandline parsing instead of rolling your own. this helps the tools react consistently to user input. Example here: <https://github.com/DFHack/scripts/blob/master/prioritize.lua#L479> and documentation here: <https://docs.dfhack.org/en/latest/docs/dev/Lua%20API.html#argparse>